Dockerfile.buf-push-action

```diff
@@ -0,0 +1,26 @@
+FROM --platform=${BUILDPLATFORM} golang:1.17.7-alpine3.15 as builder
```

Comment:
Not sure how GitHub Actions work here, I assume we need Docker?

Reply:
We don't need docker for the action. This was part of what I added from makego.

Reply:
You can remove these then, and the `docker.mk` import